typescript/sdk/src/token/adapters/ITokenAdapter.ts

```diff
@@ -53,6 +55,7 @@ export interface IHypTokenAdapter<Tx> extends ITokenAdapter<Tx> {
   quoteTransferRemoteGas(
     destination: Domain,
     sender?: Address,
+    amount?: string,
```

Comment:
`quoteTransferRemoteGas` returns the amount the user will receive on the destination chain. In order to provide this information, Khalani needs to know the amount that will be sent.